### ISSUE TITLE
space-after-keywords allow enter

### ISSUE THREAD

**Issue by czjvic** [eslint/eslint#3198]
Following code is allowed by *space-after-keywords* in *always* mode
````javascript
if
(a == 1) {
    console.log(a);
}
````

Expected force
````javascript
if (a == 1) {
    console.log(a);
}
````

In plugin should be check for count of spaces between. Currently there is only:
````javascript
hasSpace = left.range[1] < right.range[0];
````

Version: 1.0.0 RC3

**Comment by eslintbot** [comment]
Thanks for the issue! We get a lot of issues, so this message is automatically posted to each one to help you check that you've included all of the information we need to help you.

Reporting a bug? Please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
1. The source code that caused the problem
1. The configuration you're using (for the rule or your entire config file)
1. The actual ESLint output complete with line numbers

Requesting a new rule? Please be sure to include:

1. The use case for the rule - what is it trying to prevent or flag?
1. Whether the rule is trying to prevent an error or is purely stylistic
1. Why you believe this rule is generic enough to be included

Requesting a feature? Please be sure to include:

1. The problem you want to solve (don't mention the solution)
1. Your take on the correct solution to problem

Including this information in your issue helps us to triage it and get you a response as quickly as possible.

Thanks!

**Comment by nzakas** [comment]
Yeah, we should double check that it's a space rather than a new line.

**Comment by btmills** [comment]
Working on this.

**Comment by mhart** [comment]
Hmmm, I think this may have been too hasty and has caused some weirdness with keywords like `else`.

For example, with `"space-after-keywords": [2, "always"]` and the following code:

```js
if (a == null)
  doThing()
else
  doOtherThing()
```
I get the error: `Keyword "else" must not be followed by a newline`

I don't think this is the intent of this rule.

**Comment by btmills** [comment]
Yikes. Can you open a new issue for that? I'm thinking we should only warn for new lines of the next token is `(`, but we should discuss that on the issue first.

**Comment by mhart** [comment]
(yeah, or `{` I imagine too)

**Comment by mhart** [comment]
(or maybe not... not sure. Anyway, filed a new issue, cheers: #3226 )
